src/pages/SongPage.tsx

```diff
@@ -41,9 +44,9 @@ const SongPage: React.FC = () => {
 
       <IonContent>
         {/* TODO: Add error handling in case of non number song Id */}
-        {isBrowser() && RenderPrevButton(+songId)}
+        {isBrowser() && !tooNarrow ? RenderPrevButton(+songId) : null}
```

Comment:
I think better to not use ternary here and just do `{isBrowser() && !tooNarrow && RenderPrevButton(+songId)}` It'll only get to the last one if the first are all true, otherwise it'd short circuit and not render it.